File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_One/Java/Program5.java

import java.io.*;


class Program5{


	public static void main(String args[])throws IOException{

		BufferedReader br  =new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Enter the number");
		int num = Integer.parseInt(br.readLine());

		int sum = (powerOf10(10,num) - (9*num) - 10)/9;

		System.out.printf("%d",sum);


	}	


	static int powerOf10(int under,int upper){
		
		while(upper!=0){
			
			under = 10*under;
			upper--;
		}

		return under;		
	}	

}	

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_One/Cpp/program5.cpp
#include<iostream>

// Note :  while taking input take value less than 10 becoz the value exceeds the integer capacity

long powerOf10(int,int);

int main(){
	
	int num ;
	printf("Enter the number\n");
	scanf("%d",&num);

	long sum = (powerOf10(10,num) - (9*num) - 10)/9;
	
	printf("%ld",sum);
}	


long powerOf10(int under,int upper ){

	while(upper!=0){
		under = 10*under;
		upper--;
	}	
	return under;
}	
